packages/@glimmer/manager/lib/internal/index.ts

```diff
@@ -213,13 +213,36 @@ export function getInternalComponentManager(
 ///////////
 
 export function hasInternalComponentManager(definition: object): boolean {
-  return getManager(COMPONENT_MANAGERS, definition) !== undefined;
+  return (
+    hasDefaultComponentManager(definition) ||
+    getManager(COMPONENT_MANAGERS, definition) !== undefined
+  );
 }
 
 export function hasInternalHelperManager(definition: object): boolean {
-  return getManager(HELPER_MANAGERS, definition) !== undefined;
+  return (
+    hasDefaultHelperManager(definition) || getManager(HELPER_MANAGERS, definition) !== undefined
+  );
 }
 
 export function hasInternalModifierManager(definition: object): boolean {
-  return getManager(MODIFIER_MANAGERS, definition) !== undefined;
+  return (
+    hasDefaultModifierManager(definition) || getManager(MODIFIER_MANAGERS, definition) !== undefined
+  );
+}
+
+function hasDefaultComponentManager(_definition: object): boolean {
```

Comment:
VS Code suggested to underscore the argument name since it's not used yet. I'm not sure if it's actually needed though, the lint script doesn't show any warnings when I don't prefix it so it just seems a VS Code only thing?

Reply:
I think we usually do that anyway, seems fine to me